subsys/bluetooth/host/settings.c

```diff
@@ -132,7 +132,7 @@ static int set_setting(const char *name, size_t len_rd, settings_read_cb read_cb
 	ssize_t len;
 	const char *next;
 
-	if (!atomic_test_bit(bt_dev.flags, BT_DEV_ENABLE)) {
+	if (!atomic_test_bit(bt_dev.flags, BT_DEV_OPEN)) {
```

Comment:
Shouldn't these be using `BT_DEV_READY` instead? I guess we don't want to perform any actions until the host is ready?